pyrgg/params.py

```diff
@@ -99,8 +99,8 @@
 SBM_ENGINE_PARAMS = {
     1: ["vertices", "- Vertices Number (n >= 0) : "],
     2: ["blocks", "- Blocks Number (k >= 0) : "],
-    3: ["intra_probability", "- Intra Block Probability (0 <= p_intra <= 1): "],
-    4: ["inter_probability", "- Inter Block Probability (0 <= p_inter <= 1): "],
+    3: ["intra_probability", "- Within Block Probability (0 <= p <= 1): "],
```

Comment:
@sadrasabouri 
Can you explain why you believe `Within` / `Between` are better than `Intra` / `Inter`?

Reply:
Intra and inter are very hard to comprehend, at least for me, but within/between is much easier.